corehq/apps/cachehq/cachemodels.py

```diff
@@ -97,14 +97,6 @@ class DomainInvitationGenerationCache(GenerationCache):
     ]
 
 
-class CommtrackConfigGenerationCache(GenerationCache):
```

Comment:
i think this is a pretty frequently accessed model/view - are we worried about the performance hit of removing caching?